pipelines/common/src/main/java/com/google/fhir/analytics/ParquetUtil.java

```diff
@@ -180,8 +254,34 @@ public synchronized void write(Resource resource) throws IOException, ProfileExc
     }
   }
 
-  private synchronized void flush(String resourceType) throws IOException, ProfileException {
-    ParquetWriter<GenericRecord> writer = writerMap.get(resourceType);
+  /**
+   * This is to write a materialized View Definition to a Parquet file. Automatically handles file
+   * creation in a directory named after the respective view (e.g., `patient_flat_view`) with names
+   * following the "output-streaming-ddddd" pattern where `ddddd` is a string with five digits.
+   */
+  public synchronized void write(Resource resource, ViewDefinition vDef)
```

Comment:
Have you added any unit-tests for this method?

Reply:
Done.